index.html

```diff
@@ -35,32 +35,32 @@
 						</a>
 					</div>
 
-					<div id="navbarBasicExample" class="navbar-menu has-text-weight-bold">
-						<div class="navbar-end" style="font-weight: 900; color: #696969">
+					<div class="navbar-menu has-text-weight-bold" id="navbarBasicExample">
+						<div class="navbar-end" style="font-weight: 900;">
 							<div class="navbar-item">
 								<div class="buttons">
-									<a class="button is-light">
+									<a class="button is-light" style="color: #696969">
 										Formatter
 									</a>
 								</div>
 							</div>
 							<div class="navbar-item">
 								<div class="buttons">
-									<a class="button is-light">
+									<a class="button is-light" style="color: #696969">
 										MLA Guidelines
 									</a>
 								</div>
 							</div>
 							<div class="navbar-item">
 								<div class="buttons">
-									<a class="button is-light">
+									<a class="button is-light" style="color: #696969">
 										About
 									</a>
 								</div>
 							</div>
 							<div class="navbar-item">
 								<div class="buttons">
-									<a class="button is-light">
+									<a class="button is-light" style="color: #696969">
```

Comment:
You can definitely apply this color with CSS to the elements you want. Alternatively, you should define a CSS class called "is-navbar-color" or "is-nav-black" or something like that and add the class to the elements you want to style instead of manually inline styling each one. That way you cut down on repeated markup. Only do inline styles for one off elements.